### DISCUSSION TITLE
hello

### DISCUSSION THREAD

**Discussion by witchfindertr** [userbradley/BeReal#29] [Q&A]
https://mobile.bereal.com/api/search/profile

{"statusCode":400,"errorKey":null,"data":null,"fields":null,"timestamp":"2023-12-11T16:26:44.739Z","requestId":"814ae86e7f3c6f8020ae02452ea3b2d8"}

Anyone know example request?

**Comment by MrAn0nym** [comment]
Probably related to [this](https://github.com/userbradley/BeReal/discussions/24)... Props to BeReal for not letting hobby modders to have a little fun
